### PR TITLE
Fix: Typo in documentation

### PR DESCRIPTION
**Pull request by codesmith-emmy**
Few minor typos in documentation are corrected.

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -32,12 +32,12 @@ Mastodon uses a standardized, open protocol to implement federation. It is calle
 
 The **fediverse** (“federated universe”) is the name for all websites that can communicate with each other over ActivityPub and the World Wide Web. That includes all Mastodon servers, but also other implementations:
 
-* Pleroma, a modular microblogging engine,
-* Pixelfed, federated image sharing platform, which lets you share and consume media posts,
-* Misskey, which includes microblogging alongside a customizable dashboard,
-* PeerTube, which lets you upload videos to channels,
-* Plume, which lets you publish longer-form articles,
-* and many more, including individual and personal websites!
+* Pleroma, a modular microblogging engine;
```
Comment (vmstan) [inline content/en/_index.md:35]: Do we really need line ending punctuation here at all? Seems inconsistent with style of other areas. Also let's revise the `*` to `-` such as in other areas.

Comment (codesmith-emmy) [reply content/en/_index.md:35]: I think we need a punctuation (particularly a semicolon) at the end of each line to separate each item sir.
It’s a list of long sentences which follow on from the first item down to the last.
The semicolon makes it easier to read them as separate items or sentences in the same list.

**Review by andypiper — CHANGES_REQUESTED** [review]
Agree with @vmstan that the list should be revised to use `-` rather than `*` and have the punctuation removed at the end of each line. Otherwise, lgtm.

**Comment by codesmith-emmy** [comment]
The list has now been revised to use - rather than * as suggested by @vmstan and agreed to by @andypiper 

Punctuation has also been removed from the end of each line of the list.

Please check if it’s all good now. Thanks

**Review by andypiper — APPROVED** [review]
thank you!

**Pull request merged**